### PR TITLE
CHeckout i 've done smth

### PR DESCRIPTION
**Pull request by allonsg**

### CODE REVIEW

**Comment by allonsg** [comment]
what a hell

File: README.pl.md [hunk 1]
```diff
@@ -40,35 +59,49 @@ Wersja produkcyjna projektu będzie automatycznie gromadzić się i deployować
...
+Status deploymentu ostatniego commitu wyświetla się na ikonie obok jego
+identyfikatora.
 
 - ** Żółty kolor** - wykonuje się zbudowanie i deployment projektu.
 - ** Zielony kolor** - deployment zakończył się sukcesem.
-- ** Czerwony kolor** - w czasie lintingu, budowania lub deplymentu pojawił się błąd.
+- ** Czerwony kolor** - w czasie lintingu, budowania lub deplymentu pojawił się
+  błąd.
 
-Więcej informacji o statusie można zobaczyć klikając na ikonkę i w wyskakującym oknie przejść do odnośnika `Details`.
```
Comment (Relixxxy) [inline README.pl.md:55]: ebobo take robuty?

File: README.pl.md [hunk 2]
```diff
@@ -40,35 +59,49 @@ Wersja produkcyjna projektu będzie automatycznie gromadzić się i deployować
...
 
-Po jakimś czasie, zazwyczaj kilku minut, działającą stronę będzie można zobaczyć pod adresem wskazanym w wyedytowanej właściwości `homepage`. Na przykład tu znajduje się odnośnik do działającej strony dla tego repozytorium
- [https://goitacademy.github.io/parcel-project-template](https://goitacademy.github.io/parcel-project-template).
-
+Po jakimś czasie, zazwyczaj kilku minut, działającą stronę będzie można zobaczyć
+pod adresem wskazanym w wyedytowanej właściwości `homepage`. Na przykład tu
+znajduje się odnośnik do działającej strony dla tego repozytorium
+[https://goitacademy.github.io/parcel-project-template](https://goitacademy.github.io/parcel-project-template).
 
-Jeżeli otwiera się pusta strona, upewnij się, że w zakładce `Console` nie ma błędów związanych z nieprawidłowymi ścieżkami do plików projektu CSS i JS (**404**). Najprawdopodobniej wprowadzona została nieprawidłowa wartość właściwości `homepage` lub skryptu `build` w pliku `package.json`.
```
Comment (Relixxxy) [inline README.pl.md:65]: neporyadok

File: README.pl.md [hunk 3]
```diff
@@ -40,35 +59,49 @@ Wersja produkcyjna projektu będzie automatycznie gromadzić się i deployować
...
+Jeżeli otwiera się pusta strona, upewnij się, że w zakładce `Console` nie ma
+błędów związanych z nieprawidłowymi ścieżkami do plików projektu CSS i JS
+(**404**). Najprawdopodobniej wprowadzona została nieprawidłowa wartość
+właściwości `homepage` lub skryptu `build` w pliku `package.json`.
 
 ## Jak to działa
 
 ![How it works](./assets/how-it-works.png)
 
-1. Po każdym pushu w gałęzi `main` repozytorium GitHub, włącza się specjalny skrypt (GitHub Action) z pliku `.github/workflows/deploy.yml`.
```
Comment (Relixxxy) [inline README.pl.md:71]: vas zvilneno!!!!

File: README.pl.md (hunk 1 above)
Comment (allonsg) [reply README.pl.md:55]: ta vse of, che dobalsya

**Review by Relixxxy — APPROVED** [review]
good, but not for our company

**Pull request merged**